### PR TITLE
Adding thehackmic.hackclub.com

### PR DESCRIPTION
**Pull request by WolfGamer2**
## Description

The Hack Mic's Website
thehackmic.hackclub.com

### CODE REVIEW

**Review by alx-alexpark — COMMENTED** [review]
Waiting for indication of HQ approval

**Comment by radioblahaj** [comment]
It's an unoffical podcast, but it should be fine to approve. It's no different than what podcast.hackclub.com was

**Review by radioblahaj — APPROVED** [review]

**Review by alx-alexpark — APPROVED** [review]
lgtm!

**Pull request merged**
